tests/functional/aws-node-sdk/test/multipleBackend/initMPU/initMPUGcp.js

```diff
@@ -0,0 +1,96 @@
+const async = require('async');
+const assert = require('assert');
+
+const withV4 = require('../../support/withV4');
+const BucketUtility = require('../../../lib/utility/bucket-util');
+const { describeSkipIfNotMultiple, gcpClient, gcpBucketMPU, gcpLocation } =
+    require('../utils');
+const { createMpuKey } =
+    require('../../../../../../lib/data/external/GCP').GcpUtils;
+
+const bucket = 'buckettestmultiplebackendinitmpu-gcp';
+const keyName = `somekey-${Date.now()}`;
+
+const skipIfNotMultipleorIfProxy = process.env.CI_PROXY === 'true' ?
```

Comment:
pretty minor: I would also capitalize the `or`

Reply:
this conditional is removed in the XML API only refactor